source/runbooks/rotating-secrets.html.md.erb

```diff
@@ -1,25 +1,33 @@
+| Name | Secret Name | Useage | Location | How to rotate | Days to rotate |
+|---|---|---|---|---|---|
+| PagerDuty Token | pagerduty_token | Used by PagerDuty Terraform to manage PagerDuty resources|AWS Secrets Manager | Contact Operations Engineering to issue a new token and update the secret. | 180 |
+| PagerDuty Modernisation Platform Team user | pagerduty_integration_keys | Used for dead-end notifications as all schedules need a user | MoJ PagerDuty Account | Use password reset process if needed | 180 |
+| Slack Webhook URL | slack_webhook_url | Used to post alarms to Slack | AWS Secrets Manager | Contact Operations Engineeering to issue a new incoming webhook for the `Modernisation Platform Alerts` custom Slack application.  Revoke the old incoming webhook and update the secret. | 180 |
+| GitHub MP CI User PAT | github_ci_user_pat | Used to create PRs etc in GitHub actions and deploy GitHub resources via Terraform | AWS Secrets Manager | Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret. | 180 |
+| GitHub MP CI User Environments Repo PAT | github_ci_user_environments_repo_pat | Used in reusable pipelines of the modernisation-platform-environments repository. This is so that the CI user can post comments in PRs, e.g. tf plan/apply output. | AWS Secrets Manager | Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret.| 180 |
+| GitHub MP CI User Password | github_ci_user_password | Used to log in and set the PAT | AWS Secrets Manager | Log in to GitHub as the user and reset the password, update the secret | 180 |
+| Environment Management | environment_management | IDs for AWS-specific resources for environment management, such as organizational unit IDs | Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret. | 180 |
+| Nuke ID List | nuke_account_ids | Account IDs to be auto-nuked on weekly basis. This secret is used by GitHub actions job nuke.yml inside the environments repo, to find the Account IDs to be nuked. |	Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret. | N/A |
+| Nuke Block List | nuke_account_blocklist | Account IDs to be excluded from auto-nuke. AWS-Nuke (https://github.com/rebuy-de/aws-nuke) requires at least one Account ID to be present in this blocklist, while it is recommended to add every production account to this blocklist. | Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret. | N/A |
+| PagerDuty user api | pagerduty_userapi_token | PagerDuty api user token | Log in as the Modernisation Platform CI User and generate a new PAT, revoke the old one and update the secret. | 180 |
```

Comment:
Could you move this to be next to the other PagerDuty secrets please?

Reply:
Done